Source/qol/statistics.cpp

```diff
@@ -0,0 +1,97 @@
+#include "qol/statistics.h"
+
+namespace devilution {
+
+Statistics myPlayerStatistics;
+std::map<std::string, std::string> statisticsFile;
+
+void SaveMonsterKillCount();
+void LoadMonsterKillCount();
+std::string GetMonsterName(uint16_t monsterID);
+void AddMonsterWeapon(std::string &monsterName, uint16_t monsterID);
+
+void InitializePlayerStatistics()
+{
+	myPlayerStatistics = {};
+	std::fill_n(MonsterKillCounts, MAXMONSTERS, 0);
+}
+
+void LoadStatisticsFromMap()
+{
+	if (statisticsFile.find("deathCount") != statisticsFile.end())
+		myPlayerStatistics.deathCount = std::stoul(statisticsFile.at("deathCount"));
+
+	if (statisticsFile.find("ingameTime") != statisticsFile.end())
+		myPlayerStatistics.ingameTime = std::stoul(statisticsFile.at("ingameTime"));
+	LoadMonsterKillCount();
+}
+
+void SaveStatisticsToMap()
+{
+	statisticsFile["deathCount"] = std::to_string(myPlayerStatistics.deathCount);
+	statisticsFile["ingameTime"] = std::to_string(myPlayerStatistics.ingameTime);
+	SaveMonsterKillCount();
+}
+
+void CalculateInGameTime()
+{
+	uint32_t ticksNow = SDL_GetTicks();
+	myPlayerStatistics.ingameTime += (ticksNow - myPlayerStatistics.ticksSubstrahend);
+	myPlayerStatistics.ticksSubstrahend = ticksNow;

```

Comment:
subtrahend

Also should we take pause in to consideration?

Reply:
> subtrahend

yeah :)

>Also should we take pause in to consideration?

idk, should we?

Reply:
Preferably not

Reply:
Ok, I will not take pause into consideration.

Reply:
just to be sure by not taking into consideration you mean:
ignore its existance and influence on game time. Or rather - not count game time when paused?